### PR TITLE
[en] Modify short_description about namespace

### PR DESCRIPTION
**Pull request by my-git9**
I think short_description should be modified synchronously with the text.

### CODE REVIEW

File: content/en/docs/reference/glossary/namespace.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ id: namespace
 date: 2018-04-12
 full_link: /docs/concepts/overview/working-with-objects/namespaces
 short_description: >
-  An abstraction used by Kubernetes to support multiple virtual clusters on the same physical cluster.
+  An abstraction used by Kubernetes to support isolation of groups of resources within a single {{< glossary_tooltip text="cluster" term_id="cluster" >}}.
```
Comment (sftim) [inline content/en/docs/reference/glossary/namespace.md:7]: We don't use shortcodes inside glossary entry short descriptions. Can you rewrite this with just the plain word “cluster”?

**Review by sftim — CHANGES_REQUESTED** [review]

**Comment by tengqm** [comment]
yup. "virtual cluster" has become a thing these days, we'd better not use it.
/lgtm

**Review by divya-mohan0209 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**